### ISSUE TITLE
[Docs] Lifecycle Management page needed for Kubernetes 

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#9006]
### Current State
https://docs.meshery.io/tasks/lifecycle-management introduces the topic of Lifecycle Management, but doesn't expand deeply or per integration.

### Desired State
Include sequence diagrams and flows for K8s registration, discovery and states.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by shivam200446** [comment]
Hey, I want to work on it assign me!

**Comment by AryanSharma9917** [comment]
Hey @leecalcote, can I work on this issue. 
We can add  `cluster states`, `discovery` and also about `pods`, `architecture`, `secrets` and `scaling a k8s cluster` and many more.

**Comment by leecalcote** [comment]
Thanks for this suggestion, @AryanSharma9917. The specifics that need to be included are how Meshery performs lifecycle management - connection states and operator provisioning.

**Comment by AryanSharma9917** [comment]
> Thanks for this suggestion, @AryanSharma9917. The specifics that need to be included are how Meshery performs lifecycle management - connection states and operator provisioning.

Okey, so can I work on this issue?

**Comment by leecalcote** [comment]
@AryanSharma9917, yes, without assignment, though, until you show cognizance of the required details needed.
